python/cuml/cuml/tests/experimental/accel/test_basic_estimators.py

```diff
@@ -186,6 +239,7 @@ def test_defaults_args_only_methods():
 
 
 def test_kernel_ridge():
+    import cupy as cp
```

Comment:
Why move this here? Maybe we should leave a comment for people from the future to explain why it can't be imported at the top of the file (or move it back if this was just for debugging)